src/types/options.ts

```diff
@@ -63,6 +63,11 @@ export interface CommonOptions {
      * @default './dist'
      */
     outputDir?: string;
+    /**
+     * generate sourcemap
+     * @params boolean |'true'|'false' | 'inline' | 'all' | 'all-inline'
+     */
+    sourcemap?: boolean |'true'|'false' | 'inline' | 'all' | 'all-inline'
```

Comment:
It is enough to include boolean
```suggestion
    sourcemap?: boolean | 'inline' | 'all' | 'all-inline'
```